CMakeLists.txt

```diff
@@ -191,6 +191,12 @@ zephyr_ld_options(
 )
 endif()
 
+if(CONFIG_NEWLIB_LIBC_NANO)
+zephyr_ld_options(
+  -specs=nano.specs
+)
+endif()
```

Comment:
Let's move this into libc/libc/newlib/CMakeLists.txt

since nano depends on newlib.